pkg/eestream/decode.go

```diff
@@ -176,7 +176,8 @@ func (dr *decodedRanger) Size() int64 {
 	return blocks * int64(dr.es.StripeSize())
 }
 
-func (dr *decodedRanger) Range(ctx context.Context, offset, length int64) (io.ReadCloser, error) {
+func (dr *decodedRanger) Range(ctx context.Context, offset, length int64) (rc io.ReadCloser, err error) {
+	defer mon.Task()(&ctx)(&err)
```

Comment:
just as a heads up, Range returns before the data actually starts being transferred, because what Range is returning is a io.ReadCloser. so in addition to this we should probably make a monkit io.ReadCloser wrapper or something that keeps track of how long a ReadCloser is open for.

Reply:
i mostly added this tracing because there are 12 different objects that implement the Range() interface and I was struggling knowing which were being called in the code for the download

Reply:
:+1: